framework/widgets/DetailView.php

```diff
@@ -205,7 +205,7 @@ protected function normalizeAttributes()
 
         foreach ($this->attributes as $i => $attribute) {
             if (is_string($attribute)) {
-                if (!preg_match('/^([\w\.]+)(:(\w*))?(:(.*))?$/', $attribute, $matches)) {
+                if (!preg_match('/^([^:]+)(:(\w*))?(:(.*))?$/', $attribute, $matches)) {
```

Comment:
@SilverFire As I said, I decided to use the same regex that's currently used for GridView. Seems like the same problem was fixed earlier for GridView only and DetailView's regex was not synchronized with these changes. That means earlier this regex was accepted. What you suggest to do? Fix it to not allow leading digits in both DetailView and GridView or just DetailView?